src/components/fields/NullField.js

```diff
@@ -1,5 +1,20 @@
-function NullField(props) {
-  return null;
+import { Component } from "react";
+import PropTypes from "prop-types";
+
+class NullField extends Component {
+  componentWillMount() {
```

Comment:
Is there a reason why you did this in componentWillMount instead of componentDidMount?

Reply:
I just forgot it was deprecated